src/pages/questions/index.jsx

```diff
@@ -6,19 +6,47 @@ import QuestionDisplay from "./QuestionDisplay";
 import ProductInformation from "./ProductInformation";
 import UserData from "./UserData";
 import { useQuestionBuffer } from "./useQuestionBuffer";
+import Divider from "@mui/material/Divider";
+
+import Stack from "@mui/material/Stack";
+import Box from "@mui/material/Box";
 
 export default function Questions() {
   const [filterState, setFilterState] = useFilterSearch();
 
-  const { buffer, answerQuestion, remainingQuestionNb, answers } = useQuestionBuffer(filterState);
+  const { buffer, answerQuestion, remainingQuestionNb, answers } =
+    useQuestionBuffer(filterState);
   const question = buffer[0] ?? null;
 
   return (
-    <>
-      <QuestionFilter filterState={filterState} setFilterState={setFilterState} />
-      <QuestionDisplay question={question} answerQuestion={answerQuestion} />
-      <ProductInformation question={question} />
-      <UserData remainingQuestionNb={remainingQuestionNb} answers={answers} />
-    </>
+    <Box sx={{ margin: "2% 5%", overflow: "hidden" }}>
+      <Stack
+        direction={{ xs: "column", sm: "column", md: "row" }}
+        spacing={{ xs: 1, sm: 2, md: 4 }}
+      >
```

Comment:
Could be simplified with 
```suggestion
      <Stack
	sx={{ margin: "2% 5%", overflow: "hidden" }}
        direction={{ xs: "column", sm: "column", md: "row" }}
        spacing={{ xs: 1, sm: 2, md: 4 }}
      >
```

I do not understand why adding a `overflow: "hidden"` on a `div` that does not have given size